shared/src/clients/whatsapp-client.class/types/webhook.ts

```diff
@@ -102,7 +102,7 @@ interface Contact {
   wa_id: WhatsAppId
 }
 
-interface Error {
+export interface Error {
```

Comment:
strongly advise renaming these if you're going to export them. `Error` is a built-in type in JS